examples/objects_core_lib/core_lib/data_layers/service/objects_service.py

```diff
@@ -6,8 +6,8 @@ class ObjectsService(Service):
     def __init__(self, data_access: ObjectsDataAccess):
         self.data_access = data_access
 
-    def get_object(self, bucket_name: str, object_name: str):
-        return self.data_access.get_object(bucket_name, object_name)
+    def get_object(self, bucket_name: str, object_name: str, key: str):
+        return self.data_access.get_object(bucket_name, object_name, key)
 
-    def set_object(self, bucket_name: str, value):
-        self.data_access.set_object(bucket_name, value)
+    def set_object(self, bucket_name: str, value, key: str):

```

Comment:
use the same order as in the `get_object` function
def set_object(self, bucket_name: str,, key: str, value):